app/javascript/react/screens/App/common/commonUtilitiesHelper.js

```diff
@@ -0,0 +1,18 @@
+const commonUtilitiesHelper = {
+  getMostRecentEntityByCreationDate: entities =>
+    entities.reduce((prev, current) => (prev.created_on > current.created_on ? prev : current)),
+
+  groupBy: (items, key) =>
```

Comment:
So this function exists in lodash, a tool we use elsewhere in v2v, https://lodash.com/docs/4.17.4#groupBy

Wonder if using ☝️ might be a lil less code

Reply:
same as above...